web/ui/module/lezer-promql/src/promql.grammar

```diff
@@ -245,14 +245,18 @@ QuotedLabelName {
 }
 
 StepInvariantExpr {
-  expr At ( NumberLiteral | AtModifierPreprocessors "(" ")" )
+  expr At ( NumberDurationLiteral | AtModifierPreprocessors "(" ")" )
 }
 
 AtModifierPreprocessors {
   Start | End
 }
 
-NumberLiteral  {
+NumberDurationLiteral  {
+ ("-"|"+")?~signed (number | inf | nan)
+}
+
+AnotherKindOfDuration {
```

Comment:
if you have another name for this one would be cool :D

Reply:
Haha! So if I understand correctly, the only difference is that one autocompletes the unit while the other does not. Maybe just make that really explicit in the name:

* `NumberDurationLiteralNoUnitCompletion` vs.
* `NumberDurationLiteralWithUnitCompletion`

Or just:

* `NumberDurationLiteral` vs.
* `NumberDurationLiteralWithUnitCompletion`

Alternatively you can go for semantic naming, meaning that one is likely just a number, the other one is definitely a duration semantically:

* `NumberDurationLiteral`
* `NumberDurationLiteralInDurationContext`

Reply:
I chose the last proposition, seems to me the more understandable. Thank you !!!